app/module/Module.scala

```diff
@@ -22,6 +22,7 @@ import play.api.libs.concurrent.Execution.Implicits._
 import play.api.libs.mailer.MailerClient
 import play.api.libs.ws.WSClient
 import daos._
+import persistence.pool1._
```

Comment:
Are we using it? Otherwise it can be removed... If it was only added for the commented line 35...